examples/spring-boot-webflux-starter/src/main/resources/config/application.yml

```diff
@@ -0,0 +1,11 @@
+spring.profiles.active: local
```

Comment:
When I added the property to [here](https://github.com/line/armeria/pull/1326/files#diff-9f44dfc8409d0632e6ce30ab22e1729cR74), the dependencies for `reactive` and `servlet` types existed together in our module at that time. 

But now it can be removed because Spring resolves the type automatically when `spring-webflux` module is only used.

So, we also do not have to specify `spring.main.web-application-type: reactive` here. I will also remove the property in the test code. Thanks! :-)